cmd/main.go

```diff
@@ -192,18 +198,48 @@ func runAccessCheck(
 	return false, nil
 }
 
+func getClientOrDie(logger echo.Logger) client.Client {
+	cfg, err := config.GetConfig()
+	if err != nil {
+		logger.Fatal(err)
+	}
+
+	cl, err := client.New(cfg, client.Options{Scheme: scheme})
+	if err != nil {
+		logger.Fatal(err)
+	}
+
+	return cl
+}
+
+func getHTTPPort() string {
+	port := "5000"
+	if val, ok := os.LookupEnv("WM_HTTP_PORT"); ok {
+		port = val
+	}
```

Comment:
If you like this can be simplified as

```suggestion
  // import "cmp"
  // const defaultHttpPort = "5000"
	port := cmp.Or(os.Getenv("WM_HTTP_PORT"), defaultHttpPort)
```